includes/blocks/example-block/edit.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { RichText } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import { editPropsShape } from './props-shape';
+
+/**
+ * Edit component.
+ * See https://wordpress.org/gutenberg/handbook/designers-developers/developers/block-api/block-edit-save/#edit
+ *
+ * @param {Object} props The block props
+ * @return {Function} Render the edit screen
+ */
+const ExampleBockEdit = (props) => {
```

Comment:
@ryanwelcher forgive me for this one - I'm not sure if our linting even supports it but for a single parameter, functions don't require parentheses.

Reply:
ah good catch!  VSCode should be formatting on save for me. Wonder if that rule is not being run?